MediaRecorder.bs

```diff
@@ -283,6 +303,14 @@ interface MediaRecorder : EventTarget {
     the value might be surpassed, not achieved, or only be achieved over a long
     period of time.</li>
 
+    <li>Constrain the configuration of |recorder| to encode using the
+    {{BitrateMode}} specified by the value of |recorder|'s {{videoBitrateMode}}
+    attribute for all video tracks |recorder| will be recording.</li>
+
+    <li>Constrain the configuration of |recorder| to encode using the
+    {{BitrateMode}} specified by the value of |recorder|'s {{audioBitrateMode}}
+    attribute for all audio tracks |recorder| will be recording.</li>
```

Comment:
What if the BitrateMode is not supported? Do we need to throw NotSupportedError the way we do earlier?

Reply:
For the current codec etc

Reply:
Good catch. If we move this step above step 8 where we throw `NotSupportedError` that might work.